files/build_templates/docker_image_ctl.j2

```diff
@@ -282,6 +282,14 @@ start() {
         --tmpfs /tmp \
 {%- endif %}
 {%- endif %}
+{%- if sonic_asic_platform == "broadcom" %}
+{%- if docker_container_name == "syncd" %}
+        -v /var/run/docker-syncd$DEV:/var/run/sswsyncd \
+{%- endif %}
+{%- endif %}
+{%- if docker_container_name == "swss" %}
+       -v /var/log/swss$DEV:/var/log/swss:rw \
```

Comment:
do we know if logrotate works if changed the place?

Reply:
Added changes for logrotation in the latest commit